kani-driver/src/concrete_playback/test_generator.rs

```diff
@@ -119,7 +126,7 @@ impl KaniSession {
     }
 
     /// Writes the new source code to a user's source file using a tempfile as the means.
-    /// Returns whether new unit test was injected.
+    /// Returns whether the unit test was already in the source code.
```

Comment:
That doesn't match the logic implemented

Reply:
`!unit_tests.is_empty()` so it returns true if some `unit_tests` were left after filtering out those that exists. These are injected, so the return value is "true" when injection occurs, and "false" when injection did no occur b/c all tests were filtered out.

Reply:
b4b33363a22a72e4eaf5acb0156e5530d165060c